src/reducers/url-state.js

```diff
@@ -358,6 +360,13 @@ const hiddenLocalTracksByPid: Reducer<Map<Pid, Set<TrackIndex>>> = (
       hiddenLocalTracksByPid.set(action.pid, hiddenLocalTracks);
       return hiddenLocalTracksByPid;
     }
+    case 'SHOW_ALL_TRACKS': {
+      const hiddenLocalTracksByPid = new Map(state);
+      for (const key of hiddenLocalTracksByPid) {
+        hiddenLocalTracksByPid.set(key[0], new Set());
+      }
+      return hiddenLocalTracksByPid;
```

Comment:
It's possible that you could just `return new Map()` here; notice this is the initial state. Have you tried it?

Reply:
I did try that, yes, but it appears that the map's being accessed with the PIDs again, so it throws an error
![Screenshot from 2021-01-19 12-09-47](https://user-images.githubusercontent.com/8143634/104997465-a14c9900-5a4f-11eb-946b-cf8b4ac2e6dc.png)

Reply:
thanks for trying!